chef/cookbooks/provisioner/libraries/repositories.rb

```diff
@@ -17,69 +17,20 @@
 class Provisioner
   class Repositories
     class << self
-      def inspect_repos(node)
```

Comment:
I understand dropping this from the crowbar perspective, but the cookbooks don't need to know about the availability anymore?